pandas/tests/io/test_sql.py

```diff
@@ -2698,6 +2700,58 @@ def test_drop_table(conn, request):
         assert not insp.has_table("temp_frame")
 
 
+@pytest.mark.parametrize("conn_name", all_connectable)
+def test_delete_rows_success(conn_name, test_frame1, request):
+    table_name = "temp_frame"
```

Comment:
Nit: Could you make `table_name` a bit more unique between these two tests? 

We have existing issue about eventually having tests consistently clean up tables they create, but in the meantime unique names will ensure your two added test won't clobber each other if one fails.

Reply:
Alright 👍 . Done !